draft-terbu-sd-jwt-vc.md

```diff
@@ -116,7 +116,7 @@ The following are non-selectively disclosable registered JWT claims that SD-JWT-
 * `exp`
     * REQUIRED. The expiry time of the Verifiable Credential after which the proof of the Verifiable Credential is no longer valid.
 * `cnf`
-    * OPTIONAL. The confirmation method can be used to verify the Holder Binding JWT of the disclosed SD-JWT.
+    * OPTIONAL. The confirmation method as defined in [@!RFC7800]. It MAY be used when Cryptographic Holder Binding is required. It MUST contain a JWK as defined in Section 3.2 of [@!RFC7800]. Other representations of the key MUST NOT be used. "kid" (Key ID) member MUST be present in the JWK. Holder Binding JWT in the Combined Format for Presentation MUST be signed by the key identified in this claim.
```

Comment:
The "OPTIONAL... MAY" does not work, and I think as @tlodderstedt mentioned, a SHOULD is in order. I suggest the following text:

```suggestion
    * RECOMMENDED when Cryptographic Holder Binding is to be supported. Contains the confirmation method as defined in [@!RFC7800]. It SHOULD contain a JWK as defined in Section 3.2 of [@!RFC7800] and in this case, the `kid` (Key ID) member MUST be present in the JWK.  For Cryptographic Holder Binding, the Holder Binding JWT in the Combined Format for Presentation MUST be signed by the key identified in this claim.
```

Reply:
@Sakurann That would work for me. If you commit @danielfett suggestions, I'd approve this PR.